noir-projects/noir-protocol-circuits/crates/types/src/abis/accumulated_data/combined_accumulated_data.nr

```diff
@@ -52,6 +52,18 @@ impl CombinedAccumulatedData {
                 revertible.unencrypted_logs_hashes
             )
         );
+        let encrypted_log_preimages_length = array_merge(
```

Comment:
It will be cheaper to loop through them directly to add up the length without creating a new array.